packages/firebase_dynamic_links/android/src/main/java/io/flutter/plugins/firebasedynamiclinks/FirebaseDynamicLinksPlugin.java

```diff
@@ -117,6 +115,11 @@ public void onMethodCall(MethodCall call, Result result) {
   }
 
   private void handleGetInitialDynamicLink(final Result result) {
+    if (registrar.activity() == null) {
+      result.success(null);
```

Comment:
I think we should use `result.error()` here. Wouldn't the user want to know that it failed because there was no activity?

Reply:
If the activity has not started, then there *is* no initial link. I think it is WAI that it returns null.